src/pages/components/dropdown/usage.mdx

```diff
@@ -44,8 +44,8 @@ functionality—dropdown, multiselect, and combo box.
 | Type                        | Purpose                                                                                                  |
 | --------------------------- | -------------------------------------------------------------------------------------------------------- |
 | [Dropdown](#dropdown)       | Allows the user to select one option from a list.                                                        |
-| [Multiselect](#multiselect) | Allows the user to select multiple options from a list and filter.                                       |
-| [Combo box](#combo box)     | Allows a user to enter any value, or select a value from a list of suggested, likely, or desired values. |
+| [Multi-select](#multiselect)| Allows the user to select multiple options from a list and filter.                                       |
```

Comment:
This is actually correct as is, @laurenmrice. No hyphen in **multiselect**, per IBM Style word usage guidelines. :)